### DISCUSSION TITLE
Will same spelling and different meaning be counted as the same morph

### DISCUSSION THREAD

**Discussion by vlashada** [mortii/anki-morphs#309] [Q&A] [answered]
How are words that are spelled the same but have different meaning handled, such as `bat` (animal/equipment for baseball) and `bow` (curved weapon/gesture of respect)?

Will both of these be the same morph or will it make different morphs based on the context? Is the algorithm for creating morphs contextualized?

**Comment by mortii** [comment, marked as answer]
> Will both of these be the same morph or will it make different morphs based on the context? Is the algorithm for creating morphs contextualized?

Depends on how good the morphemizer you use is. If you use the `AnkiMorphs: Simple Space Splitter` or `AnkiMorphs: SSS + Punctuation` morphemizers, then no. The other morphemizers like `spaCy` should be able to differentiate between them, although no morphemizer is perfect.
